### PR TITLE
Add unmatched quote to factors-of-a-mersenne-number

### PR DESCRIPTION
**Pull request by melusc**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

~~They are not all in the same language folder but in the same language.~~

All I changed was adding an unmatched quote.

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/factors-of-a-mersenne-number.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ assert.equal(check_mersenne(3), 'M3 = 2^3-1 is prime');
 assert.equal(check_mersenne(23), 'M23 = 2^23-1 is composite with factor 47');
 ```
 
-`check_mersenne(929)` should return "M929 = 2^929-1 is composite with factor 13007
+`check_mersenne(929)` should return "M929 = 2^929-1 is composite with factor 13007"
```
Comment (RandellDawson) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/factors-of-a-mersenne-number.md:85]:
```suggestion
`check_mersenne(929)` should return the string `M929 = 2^929-1 is composite with factor 13007`
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@melusc We are actually planning to remove quotation marks for representing strings in returned values for translation purposes.  Instead, see the following on how all the test cases should be for this challenge.  Would you mind replacing the rest of them?

Thank you.

**Comment by melusc** [comment]
I replaced the rest and added a fullstop on line 85 that was missing.

**Comment by gikf** [comment]
Hey @melusc 

This PR seems to make similar changes as the existing PR #41280. As such, we are going to close this as duplicate.

If you feel you have additional changes to expand upon this PR, please feel free to push your commits and request this PR be reopened.

Thanks again! 😊

---

If you have any questions, feel free to ask questions on the ['Contributors' category on our forum](https://forum.freecodecamp.org/c/contributors) or [the contributors chat room](https://chat.freecodecamp.org/channel/contributors).

**Pull request closed without merging**